src/ai/backend/common/events.py

```diff
@@ -200,6 +200,7 @@ class KernelLifecycleEventReason(str, enum.Enum):
     EXEC_TIMEOUT = "exec-timeout"
     FAILED_TO_START = "failed-to-start"
     FORCE_TERMINATED = "force-terminated"
+    HANGTIME_EXCEEDED = "hangtime-exceeded"
```

Comment:
How about renaming this to `HANG_TIMEOUT`, just like `IDLE_TIMEOUT`?

Reply:
Resolved: 59622bdb036db0d035ffa4b130f3e541dfc94f98